src/commands/user/lookup.ts

```diff
@@ -1,13 +1,66 @@
-import { Message } from 'discord.js'
-
-// Why "_aruments" and not "arguments"?
-// Because typescript was going nuts over the fact that "arguments" is a thing, I dont know why.
-export function lookUp(
-	message: Message,
-	_arguments: any[]
-): void | Promise<Message<boolean>> {
-	if (!_arguments)
-		return message.channel.send('We require the username argument!');
-	
-	
-}
+import fetch from 'node-fetch';
+import { Message, MessageEmbed, MessageSelectMenu } from 'discord.js'
+
+var URLPattern = new RegExp('^(https?:\\/\\/)?'+
+    '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|'+
+    '((\\d{1,3}\\.){3}\\d{1,3}))'+
+    '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*'+
+    '(\\?[;&a-z\\d%_.~+=-]*)?'+
+    '(\\#[-a-z\\d_]*)?$','i');
+
+export async function lookUp(message: Message, _arguments: string[]) {
+
+	let apiURL = `https://api.polytoria.com/v1/games/info?id`;
```

Comment:
_arguments[1] should be:
```
username
```
or
```
id
```
Use switch cases to select the API to use, do not use URL.